src/drivers/mct/cime_config/config_component.xml

```diff
@@ -542,7 +542,7 @@
   <entry id="BATCH_SYSTEM">
      <type>char</type>
      <default_value>none</default_value>
-     <valid_values>moab,pbs,lsf,slurm,cobalt,cobalt_theta,none</valid_values>
+     <valid_values>lc_slurm,moab,pbs,lsf,slurm,cobalt,cobalt_theta,none</valid_values>
```

Comment:
Why do we need lc_slurm?   Was sufficient effort made to use the slurm BATCH_SYSTEM definition?